### ISSUE TITLE
[Feature]: Exrename

### ISSUE THREAD

**Issue by weirongxu** [weirongxu/coc-explorer#10]

**Comment by zgpio** [comment]
Sorry, what is the `exrename` feature specifically meaning?

**Comment by weirongxu** [comment]
https://github.com/ivanmaeder/vimv
https://github.com/laurent22/massren

**Comment by thisguychris** [comment]
Hey @weirongxu 👋

Just stumble upon this coc plugin! And I love it! I hope you consider this feature! I like the `vimv` approach where you have the full power of vim in editing the names.

It would be cool that there would be an option for the full path and just the names for relative folder (much like `vimv`)

**Comment by weirongxu** [comment]
> Hey @weirongxu 👋
> 
> Just stumble upon this coc plugin! And I love it! I hope you consider this feature! I like the `vimv` approach where you have the full power of vim in editing the names.
> 
> It would be cool that there would be an option for the full path and just the names for relative folder (much like `vimv`)

Yes, I haven't had much time to implement it now. In vim, we can even use `virtualtext` or `floatwin` to display the original filename.
